README.md

```diff
@@ -185,6 +185,8 @@ cd firmware
 # Apply address patch
 git apply unix/linux_addr.patch
 
+cd unix
```

Comment:
remove and instead change below to `pushd external/micropythohn`. Changing dir to unix here would break cmds below